lms/djangoapps/verify_student/management/commands/update_expiration_date.py

```diff
@@ -92,12 +88,7 @@ def bulk_update(self, verification_ids):
         It updates the expiration_date and sets the expiry_date to NULL for all the
         verifications whose ids lie in verification_ids
         """
-        recent_verification_qs = SoftwareSecurePhotoVerification.objects.filter(pk__in=verification_ids)
-        recent_verification_qs.update(expiration_date=F('expiry_date'))
-        recent_verification_qs.update(expiry_date=None)
-
-    def find_recent_verification(self, model, user_id):
-        """
-        Returns the most recent approved verification for a user
-        """
-        return model.filter(user_id=user_id).latest('updated_at')
+        verification_qs = SoftwareSecurePhotoVerification.objects.filter(pk__in=verification_ids)
+        verification_qs.update(expiration_date=F('updated_at') + timedelta(
```

Comment:
Should this be `created_at` instead of `updated_at`? I worry that it will just update each entry from the date that the `expiration_date` was introduced, so they might still be incorrect.

Reply:
Got it. Yeah, let me update that.